src/main/java/seedu/duke/commands/updates/UpdateMemberCommand.java

```diff
@@ -0,0 +1,45 @@
+package seedu.duke.commands.updates;
+
+import seedu.duke.Duke;
+import seedu.duke.Ui;
+import seedu.duke.commands.Command;
+import seedu.duke.commands.CommandResult;
+import seedu.duke.exceptions.DukeException;
+import seedu.duke.exceptions.parserexceptions.InvalidBudgetException;
+import seedu.duke.items.Task;
+
+public class UpdateMemberCommand extends Command {
+
+    private final int memberToBeAdded;
+    private final int memberToBeRemoved;
+    private final Task taskToBeUpdated;
+    private final String flag;
+
+    public UpdateMemberCommand(String flag, int memberToBeAdded, Task taskToBeUpdated, int memberToBeRemoved) {
+        this.memberToBeAdded = memberToBeAdded;
+        this.memberToBeRemoved = memberToBeRemoved;
+        this.taskToBeUpdated = taskToBeUpdated;
+        this.flag = flag;
+    }
+
+    public CommandResult execute() throws DukeException, InvalidBudgetException {
+        if (flag.equalsIgnoreCase("add")) {
+            addMember(memberToBeAdded);
+        } else if (flag.equalsIgnoreCase("change")) {
```

Comment:
A check needs to be performed if the `memberToBeRemoved` is the same as the `memberToBeAdded`.